packages/http/src/validator/validators/security/__tests__/security.spec.ts

```diff
@@ -18,7 +18,15 @@ describe('validateSecurity', () => {
 
   describe('when security scheme uses Basic authorization', () => {
     const securityScheme: HttpSecurityScheme[][] = [
-      [{ id: faker.random.word(), scheme: 'basic', type: 'http', key: 'sec' }],
+      [
+        {
+          id: faker.random.word(),
+          scheme: 'basic',
+          type: 'http',
+          key: 'sec',
+          extensions: { 'x-test': faker.random.word() },
```

Comment:
I'm not sure if this is a relevant question here, but are all of these tests directly affected by the addition of `extensions` here, in the sense that what they're testing for has changed?

Reply:
Yes - when I upgraded the types version, the type `HttpSecurityScheme` had `extensions` added as a new required filed (see this pr: https://github.com/stoplightio/types/pull/126). Since it is required, then yes I can see that it kinda of changes what they're testing for. Do you think we need to add specific tests around the new required field or are you thinking of this more as a breaking change then?